src/Core/Graphics/LevelSprite.cpp

```diff
@@ -122,6 +123,11 @@ namespace obe::Graphics
         m_layerChanged = true;
     }
 
+    void LevelSprite::setAntiAliasing(bool antiAliasing)
```

Comment:
It would be nice to have a `getAntiAliasing` method to see if it's enabled or not :)

Reply:
Also the `setAntiAliasing` and `getAntiAliasing` methods should be added to the Lua bindings